engine-nextgen/src/main/kotlin/graphql/nadel/enginekt/transform/hydration/batch/NadelBatchHydrationInputBuilder.kt

```diff
@@ -140,13 +148,6 @@ internal object NadelBatchHydrationInputBuilder {
             .asSequence()
             .map { it.value }
             .flatten(recursively = true)
-            .let {
-                if (filterNull) {
-                    it.filterNotNull()
-                } else {
-                    it
-                }
-            }
```

Comment:
Why is filterNotNull no longer needed ??

Reply:
I wanted to remove it and introduce `.filterNotNull()` in the call site where appropriate, but it's not necessarily needed so I rolled it back